CONTRIBUTING.md

```diff
@@ -17,15 +17,19 @@ The TL;DR is: **don't clone your fork**, and it matters where on your filesystem
 
 If you don't care how and why and just want something that works, follow these steps:
 
-1. [fork this repo][fork]
+1. [fork this repo on the Github webpage][fork]
 1. `go get github.com/exercism/cli/exercism`
 1. `cd $GOPATH/src/github.com/exercism/cli` (or `cd %GOPATH%/src/github.com/exercism/cli` on Windows)
-1. `git remote set-url origin https://github.com/<your-github-username>/cli`
+1. `git remote rename origin upstream`
+1. `git remote add origin git@github.com:<your-github-username>/cli.git`
+1. `git checkout -b development`
+1. `git push -u origin development` (setup where you push to, check it works)
 1. `go get -u github.com/golang/dep/cmd/dep`
    * depending on your setup, you may need to install `dep` by following the instructions in the [`dep` repo](https://github.com/golang/dep)
 1. `dep ensure`
+1. `git update-index --assume-unchanged Gopkg.lock` (prevent your dep changes being committed)
```

Comment:
😍 I didn't know about this. It's so good!

Reply:
:-)

Reply:
👍